middlewares/x-powered-by/CHANGELOG.md

```diff
@@ -12,3 +12,9 @@
 - Excluded some files from npm package
 
 Changes in versions 1.0.0 and below can be found in [Helmet's changelog](https://github.com/helmetjs/helmet/blob/master/CHANGELOG.md).
+
+### Changed
```

Comment:
This should be at the top of the changelog under a new "Unreleased" heading. It should roughly have this shape:

```md
# Changelog

## Unreleased

### Changed

- Removed `setTo` option. ...

## 1.1.0 - 2019-05-26

...
```